packages/schematics/angular/application/files/src/test.ts.template

```diff
@@ -7,6 +7,7 @@ import {
   platformBrowserDynamicTesting
 } from '@angular/platform-browser-dynamic/testing';
 
+// tslint:disable-next-line: no-any
 declare const require: any;
```

Comment:
Can you define the type for this instead?  Tslint will be phased out moving forward and the less tslint comments that need to be migrated, the better.

In this case, the Webpack-only `require.context` function is all that is needed: https://webpack.js.org/api/module-methods/#requirecontext

Reply:
@clydin Done.